src/components/Checkout/Checkout.js

```diff
@@ -1,208 +1,200 @@
-													<Input
-														id='number'
-														placeholder='(987) 654 -3210'
-														type='text'
-													
-													></Input>
-												</FormControl>
-											</GridItem>
-											<GridItem colSpan={2}>
-												<Heading size='xl'>Shipping Details</Heading>
-											</GridItem>
-											<GridItem colSpan={2}>
-												<FormControl isRequired>
-													<FormLabel htmlFor='street-address'>
-														Street address
-													</FormLabel>
-													<Input
-														id='street-address'
-														placeholder='Street address'
-														type='text'
-														
-													></Input>
-												</FormControl>
-											</GridItem>
-											<GridItem colSpan={2}>
-												<FormControl isRequired>
-													<FormLabel htmlFor='building-number'>
-														Building, floor, apt, suite
-													</FormLabel>
-													<Input
-														id='building-number'
-														placeholder='Building, floor, apt, suite'
-														type='text'
-														
-													></Input>
-												</FormControl>
-											</GridItem>
-											<GridItem colSpan={2}>
-												<SimpleGrid
-													columns={3}
-													rowGap={6}
-													columnGap={3}
-													w='full'
-												>
-													<GridItem colSpan={1}>
-														<FormControl isRequired>
-															<FormLabel htmlFor='city'>City</FormLabel>
-															<Input
-																id='city'
-																placeholder='City'
-																type='text'
-																
-															></Input>
-														</FormControl>
-													</GridItem>
-													<GridItem colSpan={1}>
-														<FormControl isRequired>
-															<FormLabel htmlFor='zipcode'>Zip Code</FormLabel>
-															<Input
-																id='zipcode'
-																placeholder='Zip Code'
-																type='text'
-																
-															></Input>
-														</FormControl>
-													</GridItem>
-													<GridItem colSpan={1}>
-														<FormControl isRequired>
-															<FormLabel htmlFor='state'>State</FormLabel>
-															<Input
-																id='state'
-																placeholder='State'
-																type='text'
-																
-															></Input>
-														</FormControl>
-													</GridItem>
-												</SimpleGrid>
-											</GridItem>
-											<GridItem colSpan={2}>
-												<Heading size='xl'>Payment Details</Heading>
-												<Text>Pay with credit card via Stripe</Text>
-											</GridItem>
-											<GridItem colSpan={2}>
-												<PaymentForm />
-											</GridItem>
-										</SimpleGrid>
-									</div>
-								</VStack>
-							</GridItem>
-							<GridItem colSpan={1}>
-								<OrderSummary />
-							</GridItem>
-						</SimpleGrid>
-					</Flex>
-				</Elements>
-			)}
-		</div>
-	);
```

Comment:
Verify that changes are not made to spacing in future PRs so that additions and deletions can be more clearly seen in the future. In this PR, only a few lines were changed, but it was hard to tell in some cases where Git does not tell you if it's spacing or actual changes.